### DISCUSSION TITLE
Proper editor for JSON(B) fields

### DISCUSSION THREAD

**Discussion by kevcodez** [supabase/supabase#601] [Feature Requests]
I'm a heavy user of JSONB, allow me to use PostgreSQL in a non-relational way.

While supabase allows adding a JSON(B) field, the editing experience is very simple.

A proper JSON editor (i.e. Monaco) with syntax highlighting, editing support (i.e. adding double quotes automatically) would be a nice addition and would certainly improve the way you can edit JSON(B) data in Supabase.

**Comment by kiwicopple** [comment]
Agreed! Good idea @kevcodez - we'll think of some improvements to our current JSON editor

**Reply by kevcodez** [reply]
Let me know if I can contribute and help out ✌️
